index.rst

```diff
@@ -73,23 +73,34 @@ EditorConfig organization.
 File Format
 ===========
 
+.. versionchanged:: 0.16.1
+
 EditorConfig files are in an INI-like file format.
-In an EditorConfig file, all beginning whitespace on each line is considered
-irrelevant. Each line must be one of the following:
+To read an EditorConfig file, take one line at a time.  For each line:
+
+#. Strip all leading and trailing whitespace.
+#. If the remaining text is empty, ignore the line.
```

Comment:
This is technically correct, but I think it's more readable if we keep the Blank bullet point.

> To read an EditorConfig file, take one line at a time. For each line, strip all leading and trailing whitespace.
>
> The types of lines are:
>
> - Blank: Contains nothing. Blank links are ignored.

What do you think?

Reply:
Sorry for this one more comment. I think we should be OK once this is resolved.

Reply:
No problem --- modified:
```diff
 EditorConfig files are in an INI-like file format.
-To read an EditorConfig file, take one line at a time.  For each line:
+To read an EditorConfig file, take one line at a time, from beginning to end.
+For each line:
 
-#. Strip all leading and trailing whitespace.
-#. If the remaining text is empty, ignore the line.
-#. If the remaining text is not empty, process the text as specified for its
-   type below.
+#. Remove all leading and trailing whitespace.
+#. Process the remaining text as specified for its type below.
 
 The types of lines are:
 
+- Blank: Contains nothing.  Blank lines are ignored.
+
 - Comment: starts with a ``;`` or a ``#``.  Comment lines are ignored.
```